tests/scenarios/test_functionality_nimbus.py

```diff
@@ -15,14 +15,14 @@
 
 
 @given("Firefox has loaded a webpage")
-def load_mozilla_wepage(navigate_using_url_bar, selenium, simplehttpserver):
-    navigate_using_url_bar("http://localhost:8000")
+def load_test_wepage(navigate_using_url_bar, selenium, static_server):
```

Comment:
Oops, missed this earlier (sorry) looks like a typo and this should be "webpage" and not "wepage"